rules/less-is-more/rule.md

```diff
@@ -0,0 +1,61 @@
+
+Here are a few helpful ways we can employ a “Less is more” approach:
+
+## Leave it out:
+
+"Just because we can does not mean we should"
+
+Including a design element, feature or product enhancement, because we have the technical ability to do so with no impact on the user experience. It could overcomplicate a simple task, cause cognitive overload and ultimately distract an audience from processing through a task with less friction.  It’s Equally as important to leave out a good idea than it is to include one, especially if it doesn’t help the user achieve their goals.
+
+![Figure: ❌ Bad example - overbranded background](microsoftteams-image-withlogos.png "Figure: Bad")
+
+![Figure: ✅ Good Example - reduced design clutter by limiting design elements](microsoftteams-image.png)
+
+Setting a bar or standard of minimum requirements for every new feature you would like to add would be a good way to measure and decide if it enhances the user experience and if it’s really necessary to add.
+
+## Keep it simple
+
+ By keeping things simple we reduce complexity and avoid the cognitive overload required for a user to make an informed decision.  We want to keep things easily digestible to as broad an audience as possible. Again mainly focus the user on the completion of a task. 
+
+Using simple descriptive language or a minimalist design approach can often demystify complicated topics and make information easily digestible so that users can make decisions with ease.
+
+![Figure: ❌ Bad example - Responding with no clear answer](responding-bad-example.jpg)
+
+![Figure: ✅ Good example - responding with a definite answer](good-example-clarity.jpg)
+
+## Empathise with the user 
+
+The word empathise means that you start to subjectively put yourself in the user’s shoes and try to experience what they are feeling.  Try to forecast how your design choices will impact other productions that might use it.
+
+Don’t forget! Most users will only invest time if it benefits them and if it is easy to process a task with minimum frustration. A large portion of users will disengage with your content if you make it harder for them to digest.
+
+![Figure: ❌ Bad example - video text over html text](text-over-text.jpg)
+
+![Figure: ✅ Good example (https://www.sanleonardo.it) - use of background video with html text](san-leonardo-website.jpg)
+
+Defining user empathy at various stages will help you develop a deeper understanding of the user's needs and ultimately help your products or assets to be used over and over again.
+
+Developing empathy maps for a user might be a handy tool to use when you are trying to empathise with the user.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/QwF9a56WFWA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
Hey @2fernandez 

See how videos are added in rules: https://ssw.com.au/rules/rule/#8-videos 

You don't need the whole <iframe> if you use the right markdown syntax

1. Please fix